src/components/coordinatetransformation/EpsgSelection.vue

```diff
@@ -64,8 +67,8 @@
  * CoordinateSelection er drop-down menuen af EPSG-koder i både input- og outputkomponenterne
  */
 import { onMounted, ref, inject } from 'vue'
-import { useStore } from 'vuex'
-import { useRoute } from 'vue-router'
+// import { useStore } from 'vuex'
+// import { useRoute } from 'vue-router'
```

Comment:
Skulle de her linjer ikke bare slettes hvis ikke der er brug for dem? Gælder også længere nede hvor der er en del udkommenterede linjer.

Reply:
Jo, de skal. 
Det var derfor, jeg skrev ovenover: 'mangler en del clean up'

Reply:
> Det var derfor, jeg skrev ovenover: 'mangler en del clean up'

Okay. En anden gang vil jeg foretrække at du tager hånd om oprydningen før du merger koden. Det er den slags man nemt glemmer at få fulgt op på og så har man lige pludseligt en masse overflødigt kode inden helt ved hvad man skal stille op med.